### ISSUE TITLE
sb-volume doesn't update mute state when headphones are inserted

### ISSUE THREAD

**Issue by warthog18** [LukeSmithxyz/voidrice#1304]
Hello to You All,
on Thinkpad X220, Artix configurd with LARBS as of april 2023, dwm updated to 6.4.
Volume controls, mute command and scroll are perfectly working with wpctl command.

Steps to reproduce:

1) without headphones plugged, set mute ON. Icon in statusbar is correctly updated
2) plug headphones: the statusbar icon remains in mute (incorrect). Play a video and you will listen to audio (correct because the audio was muted ONLY without headphones). If you push the volume keys, volume icon is updated.
3) unplug headphones: icon in statusbar remains in mute, and the audio is muted (correct)

To summarize: the hardware is working correctly (it retains the state mute/unmute without/with headphones respectively).
The issue is the update of the icon in the statusbar....

Any clue? Thanks a lot!
